eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -349,6 +349,56 @@ public function testLoadVersionInfoCacheIsMiss()
         $handler->loadVersionInfo(2, 1);
     }
 
+    /**
+     * @covers \eZ\Publish\Core\Persistence\Cache\ContentHandler::loadVersionInfo
+     */
+    public function testLoadVersionInfoWithoutVersionNumberCacheIsMiss()
+    {
+        $this->loggerMock->expects($this->once())->method('logCall');
+        $cacheItemMock = $this->getMock(ItemInterface::class);
+        $this->cacheMock
+              ->expects($this->once())
+              ->method('getItem')
+              ->with('content', 'info', 2, 'versioninfo', 0)
```

Comment:
This might be my preference, but I would change this line to:

```suggestion
              ->with('content', 'info', 2, 'versioninfo', ContentHandler::PUBLISHED_VERSION)
```

We are expecting that `getItem` method will be invoked with `\eZ\Publish\Core\Persistence\Cache\ContentHandler::PUBLISHED_VERSION` (whatever it means) as 4th argument, right? :-)